components/list/examples/10-appearance.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { List, ListItem } from '../src';
+import { listGenerator } from './_utils';
+
+export default () => (
+	<>
+		<h3>Link List</h3>
+		<List appearance="link">
```

Comment:
`appearance` prop should be used consistently across our components for the primary/hero/neutral/faint values. 

Perhaps we use `type` as the prop? Is type valid... since it's an html attribute?